collection_requirements.rst

```diff
@@ -38,6 +38,7 @@ Collection Infrastructure
 
 * MUST have a publicly available issue tracker, that does not require a paid level of service to create an account or view issues.
 * MUST use the `Ansible Code of Conduct (CoC) <https://docs.ansible.com/ansible/latest/community/code_of_conduct.html>`_ by copying `CODE_OF_CONDUCT.md <https://raw.githubusercontent.com/ansible-collections/.github/main/CODE_OF_CONDUCT.md>`_ into the root of your Git repository
+* MUST be published to `Ansible Galaxy <https://galaxy.ansible.com>_.
```

Comment:
Should this be here or towards the end in "Requirements for collections to be included in the Ansible Package" or both ?

Reply:
I think it is fine here, since it should also be a requirement for any serious collection that's not for internal use.

Reply:
2020-12-09 IRC meeting we said we were OK with this as is